tests/vexflow_test_helpers.ts

```diff
@@ -200,6 +210,10 @@ export class VexFlowTests {
     Petaluma: ['Petaluma', 'Gonville', 'Bravura', 'Custom'],
     Leland: ['Leland', 'Bravura', 'Custom'],
     MuseJazz: ['MuseJazz'],
```

Comment:
During testing / migration, we should remove the fallback fonts in the font stacks.

```ts
    Bravura: ['Bravura'/*, 'Custom'*/],
    Gonville: ['Gonville'/*, 'Bravura', 'Custom'*/],
    Petaluma: ['Petaluma'/*, 'Gonville', 'Bravura', 'Custom'*/],
    Leland: ['Leland'/*, 'Bravura', 'Custom'*/],
```

That way, we can easily see which glyphs are missing from each font.

Reply:
We can do that once Glyph goes away. Otherwise we will get a RunTime error.